test/jdkOptsTest/JDKOptsTest.gmk

```diff
@@ -29,14 +29,14 @@ $(BUILDTESTDIR)/JDKOptsTest.ok: \
 	    $(TESTDIR)/jdkOptsTest/JDKOptsTest.java \
 	    $(JTREG_IMAGEDIR)/lib/jtreg.jar \
 	    $(JTREG_IMAGEDIR)/bin/jtreg
-	$(MKDIR) -p $(@:%.ok=%)/classes
+	$(MKDIR) $(@:%.ok=%)/classes
 	$(JDKJAVAC) \
 		-d $(@:%.ok=%)/classes \
 		-cp $(JTREG_IMAGEDIR)/lib/jtreg.jar \
 		-Xlint -Werror \
 		-encoding ASCII $(TESTDIR)/jdkOptsTest/JDKOptsTest.java
 	$(RM) -rf $(@:%.ok=%)/tmp
```

Comment:
`$(RM)` is already defined to include the `-rf`, so suggest you remove those options here

Reply:
Good catch. I only searched for `$(RM) -f` patterns before. Searching for `$(RM) -rf` hit two locations, both addressed in https://github.com/openjdk/jtreg/pull/259/commits/167328701a7f253b0eadbfe638545063fb217e5e

Reply:
By the way, no hits for `$(RM) -fr` ... 😉